### PR TITLE
#2407: Prefer embedded MP4-metadata for episode and season numbers

### PR DESCRIPTION
**Pull request by viaregio**
**Changes**
New `LbraryOption` `EnableEmbeddedEpisodeInfos` for TV-Shows.

**Issues**
#2407

### CODE REVIEW

**Comment by Artiume** [comment]
https://github.com/jellyfin/jellyfin/pull/2421

File: src/strings/en-gb.json [hunk 1]
```diff
@@ -597,6 +597,8 @@
     "PreferredNotRequired": "Preferred, but not required",
     "PreferEmbeddedTitlesOverFileNamesHelp": "This determines the default display title when no internet metadata or local metadata is available.",
     "PreferEmbeddedTitlesOverFileNames": "Prefer embedded titles over filenames",
+    "PreferEmbeddedEpisodeInfosOverFileNamesHelp": "This uses the episode informations from the embedded metadata if available.",
+    "PreferEmbeddedEpisodeInfosOverFileNames": "Prefer embedded episode informations over filenames",
```
Comment (JustAMan) [inline src/strings/en-gb.json]: If en-gb version doesn't differ from en-us you don't need to specify it, only specify translation parts which differ.

Comment (dkanada) [reply src/strings/en-gb.json]: Well we already have so many duplicate strings in the British translation I don't think it matters at this point. I assume the same thing happens for all similar languages.

Comment (JustAMan) [reply src/strings/en-gb.json]: That doesn't mean we should keep piling dupes tho ;)

**Pull request merged**